ping_one/manifest.json

```diff
@@ -2,20 +2,25 @@
   "manifest_version": "2.0.0",
   "app_uuid": "5fce4e84-ab6e-4ba6-b996-ac85e134642e",
   "app_id": "ping-one",
-  "display_on_public_website": false,
+  "display_on_public_website": true,
   "tile": {
     "overview": "README.md#Overview",
     "configuration": "README.md#Setup",
     "support": "README.md#Support",
     "changelog": "CHANGELOG.md",
-    "description": "Gain insights into PingOne logs",
-    "title": "ping_one",
-    "media": [],
+    "description": "Gain insights into PingOne logs.",
+    "title": "ping-one",
```

Comment:
<img width="642" alt="image" src="https://github.com/user-attachments/assets/4c361389-01d4-4d51-b762-d8035a6f465a">

I'm sorry, I was wrong. Please change it back to `PingOne`

Reply:
@Wyrine I have reverted it back